src/query/codegen/src/writes/register.rs

```diff
@@ -75,7 +75,7 @@ pub fn codegen_register() {
             "
                 pub fn register_{n_args}_arg<{arg_generics_bound} O: ArgType, F, G>(
                     &mut self,
-                    name: &'static str,
+                    name: &str,
```

Comment:
cc @RinChanNOWWW, we don't need to play with `&'static str` anymore.

Reply:
LGTM!